e2e_samples/parking_sensors/README.md

```diff
@@ -291,46 +291,23 @@ Set up the environment variables as specified, fork the GitHub repository, and l
    - To further customize the solution, set parameters in `arm.parameters` files located in the `infrastructure` folder.
       - To enable Observability and Monitoring components through code(Observability-as-code), please set enable_monitoring parameter to true in  `arm.parameters` files located in the `infrastructure` folder. This will deploy log analytics workspace to collect monitoring data from key resources, setup an Azure dashboards to monitor key metrics and configure alerts for ADF pipelines.
   
-  **Login**
+  **Login and Cluster Configuration**
 
-- Ensure that:
-  - You have loaded your environment file. To do so run:
-
-        ```bash
-        source .devcontainer/.env
-        ```
-
-  - You are logged in to the Azure CLI. To login, run
-
-        ```bash
-        az config set core.login_experience_v2=off
-        az login --tenant $TENANT_ID
-        az config set core.login_experience_v2=on
-        ```
-
-  - Azure CLI is targeting the Azure Subscription you want to deploy the resources to. To set target Azure Subscription, run
-
-        ```bash
-        az account set -s $AZURE_SUBSCRIPTION_ID
-        ```
-
-  - (Skip this if you are using our Dev Container) Azure CLI is targeting the Azure DevOps organization and project you want to deploy the pipelines to. To set target Azure DevOps project, run
-
-        ```bash
-        az devops configure --defaults organization=$AZDO_ORGANIZATION_URL project=$AZDO_PROJECT
-        ```
+- Ensure that you have completed the configuration for the variables described in the previous section, titled **Configuration: Variables and Login**.
 
+  - This configuration will be used during the environment deployment process to facilitate login.
   - Create a `cluster.config.json` Spark configuration from the [`cluster.config.template.json`](./databricks/config/cluster.config.template.json) file. For the "node_type_id" field, select a SKU that is available from the following command in your subscription:
 
-        ```bash
-        az vm list-usage --location "<YOUR_REGION>" -o table
+  -     ```bash
```

Comment:
I see you were trying to fix the indentation of the feature branch but  if you preview the file on your branch shows like bellow - is still not fixed. Can you please fix it? In the main branch it is showing still correct if you want to copy from there.
![image](https://github.com/user-attachments/assets/3bac829f-c202-4ad5-939e-5a30fd64d642)

Reply:
definitely it was not right. Fixed